src/main/js/wallet/MoneroWallet.js

```diff
@@ -658,15 +658,15 @@ class MoneroWallet {
    * 
    * @param {MoneroTxConfig|object} config - configures the transaction to create (required)
    * @param {string} config.address - single destination address (required unless `destinations` provided)
-   * @param {BigInteger|string} config.amount - single destination amount (required unless `destinations` provided)
-   * @param {int} config.accountIndex - source account index to transfer funds from (required)
-   * @param {int} config.subaddressIndex - source subaddress index to transfer funds from (optional)
-   * @param {int[]} config.subaddressIndices - source subaddress indices to transfer funds from (optional)
-   * @param {boolean} config.relay - relay the transaction to peers to commit to the blockchain (default false)
-   * @param {MoneroTxPriority} config.priority - transaction priority (default MoneroTxPriority.NORMAL)
+   * @param {BigInt|string} config.amount - single destination amount (required unless `destinations` provided)
```

Comment:
We should remove string support since using native BigInt (I can do this later in the proofing process).